src/SFA.DAS.EAS.Web/Controllers/EmployerManageApprenticesController.cs

```diff
@@ -109,30 +109,30 @@ public class EmployerManageApprenticesController : BaseController
             var validatorResult = ModelState.IsValid ? await _orchestrator.ValidateApprenticeshipStopDate(hashedAccountId, hashedApprenticeshipId, model) : null;
             var validationFailed = validatorResult != null && validatorResult.Count > 0;
 
+            var userId = OwinWrapper.GetClaimValue(@"sub");
+
             if (ModelState.IsValid && !validationFailed)
             {
-                var userInfo = GetExtractUserClaims();
-
-                var statusChangeModelWithNewStopDate = new ChangeStatusViewModel()
+                var statusChangeModelWithNewStopDate = new ChangeStatusViewModel

```

Comment:
Should UpdateApprenticeshipStopDate use a ChangeStatusViewModel? This sounds like a VM for something else, and has properties like ChangeType= ChangeStatusType.Stop, which seems meaningless in this context.

Reply:
Are we creating a viewmodel to pass to the orchestrator? Seems wrong. Some of those properties seem really odd anyway, like ChangeConfirmed=true?

Reply:
Am changing to ChangeStopDateViewModel